proto/src/main/proto/Rss.proto

```diff
@@ -605,12 +605,13 @@ message RssReassignFaultyShuffleServerRequest{
   int32 shuffleId  = 1;
   repeated int32 partitionIds = 2;
   string faultyShuffleServerId = 3;
+  repeated int32 needLoadBalancePartitionIds = 4;
 }
 
 message RssReassignFaultyShuffleServerResponse{
   StatusCode status = 1;
-  ShuffleServerId server = 2;
-  string msg = 3;
+  string msg = 2;
+  ShuffleHandleInfo handle = 3;
```

Comment:
Maybe we should return a reassignment instead of `ShuffleHandleInfo` here?

Reply:
This is another point I want to explain. I hope the reassign could return the latest assignment rather than a replacement server if the reassignment is a replacement server you said.

If the reassignment is the task owned latest assignment including the latest partitions -> servers, I think it's good. But I'm not sure is this any difference with the current latest shuffle handle info which is readonly for client side.